installer/bundle/bundle_skel.sh

```diff
@@ -610,36 +610,17 @@ pkg_upd() {
         return $?
     else
         [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
-        rpm --upgrade --replacepkgs $FORCE ${pkg_filename}.rpm
+        # Only for RedHat 7.5 call upgrade with --replacepkgs flag and only if force flag is not set
```

Comment:
Can you add a comment here explaining why RH 7.5 needs a special case here? Do we know this is specific to RH 7.5 or could other similar major.minor versions (e.g. 7.4, 7.6, 8,0, etc...) need this change too?

Reply:
@AaronRM  added . It was potential fix for an issue reported only for redhat 7.5 (but the issue itself was not reproducible locally, only in customer env). I would have removed it completely, but because we never got to the actual root cause and whether this fix really resolved the issue, I feel we should leave it here for now. Eventually , we will remove it for red 7.5 too if customer see issues after we start using it with only 7.5